app/bundles/CampaignBundle/Executioner/EventExecutioner.php

```diff
@@ -190,6 +191,16 @@ public function executeForContacts(Event $event, ArrayCollection $contacts, Coun
         }
 
         $config = $this->collector->getEventConfig($event);
+
+        static $jump;
```

Comment:
I still need to test this but a note. Let's not use a static in a method. PHP will keep that static across all instantiation of objects which can make testing a nightmare when you're trying to hunt down why they are getting unexpected results. Use a class property instead.

Reply:
I reworked it in more readable flow